sys/include/net/gnrc/tx_sync.h

```diff
@@ -34,15 +35,16 @@ extern "C" {
 /**
  * @brief   TX synchronization data */
 typedef struct {
-    mutex_t signal;     /**< Mutex used for synchronization */
+    mutex_t signal;             /**< Mutex used for synchronization */
+    netdev_tx_info_t tx_info;   /**< auxiliary data about transmission */
 } gnrc_tx_sync_t;
 
 /**
  * @brief   Helper to initialize a gnrc_tx_sync_t structure
  */
 static inline gnrc_tx_sync_t gnrc_tx_sync_init(void)
 {
-    gnrc_tx_sync_t result = { .signal = MUTEX_INIT_LOCKED };
+    gnrc_tx_sync_t result = { .signal = MUTEX_INIT_LOCKED, .tx_info = { .flags = 0 } };
```

Comment:
Why need to set fields explicitly to 0?

Reply:
Good question. Maybe I didn't knew that initializer lists will zero out non-specified members back then?